src/popup/router/pages/Transactions.vue

```diff
@@ -103,12 +103,15 @@ export default {
       }
     },
     async getLatest() {
+      if (this.loading) return;
+      this.loading = true;
       const transactions = await this.$store.dispatch('fetchTransactions', {
         limit: 10,
         page: 1,
         recent: true,
       });
       this.updateTransactions(transactions);
+      this.loading = false;
```

Comment:
Would be more accurate to wrap it into `try`/`finally` to avoid freezing the loading state if the request got failed for some reason.